WooCommerce/WooCommerceTests/Authentication/AuthenticationManagerTests.swift

```diff
@@ -209,8 +210,7 @@ final class AuthenticationManagerTests: XCTestCase {
     func test_it_can_display_jetpack_error_for_org_site_credentials_sign_in() {
         // Given
         let manager = AuthenticationManager()
-        let testSite = "http://test.com"
-        let siteInfo = WordPressComSiteInfo(remote: ["isWordPress": true, "hasJetpack": false, "urlAfterRedirects": testSite])
+        let siteInfo = WordPressComSiteInfo(remote: ["isWordPress": true, "hasJetpack": false, "urlAfterRedirects": testSiteURL])
```

Comment:
This test failed 2 out of 2 times in CI 🤔 

Maybe there's more work I need to do. Reverting to draft.